### ISSUE TITLE
Clearing field with cross button in Edge does not work for many2one fields

### ISSUE THREAD

**Issue by yostashiro** [odoo/odoo#39922]
### Impacted versions:

Checked on 12.0 and 13.0.  Probably others as well.

### Steps to reproduce:

Access Odoo using Edge (Microsoft Edge 44.18362.387.0).
Press the cross symbol that appear on the right inside the field for a many2one field.

### Current behavior:

It appears that the field value is cleared when cross is pressed, but the value comes back upon save. 

### Expected behavior:

The cleared value should not come back upon save.

### Video/Screenshot link (optional):
![24902A50-A952-4FB1-91B6-628F804D5FAC](https://user-images.githubusercontent.com/7766939/68372436-31246100-0184-11ea-9b6f-30bb28c62c4b.GIF)

**Comment by pedrobaeza** [comment]
That cross button is something specific of Edge AFAIK. And it's the same as Chrome autocomplete feature: both are not compatible with Odoo itself.

**Comment by yostashiro** [comment]
Yes, it would at least be good to know if this can be taken care of by Odoo since some users get confused and it's prone errors.

**Comment by pedrobaeza** [comment]
I don't know if it can be deactivated, but why not using recommended browsers Chrome/Firefox?

**Comment by yostashiro** [comment]
That would be my suggestion too, but I think there tend to be more users in Japan who are locked to use IE/Edge due to whatever reason.

And the recommendation of Chrome/Firefox is not obvious here http://www.odoo.com/documentation/13.0/setup/deploy.html#supported-browsers.  Maybe this document can be updated with a side note.

**Comment by pedrobaeza** [comment]
These are my "recommended browsers", not official ones, hehe. Any way, as stated, Chrome also has the problem with autocomplete feature, which I say to my users that they have to disable for optimal use with Odoo.

Let's wait for any answer from Odoo about the possibility of supporting this.

@mart-e you are my wildcard!

**Comment by mart-e** [comment]
I am not sure if it is easy to detect autofill and autoclear. I guess it depends how the different browsers implemented it.
That would be for the web team, @aab-odoo maybe ?

**Comment by C3POdoo** [comment]
Dear @yostashiro,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._
